### PR TITLE
Updated release notes in preparation for 2.4 final

### PR DESCRIPTION
**Pull request by degville**

### CODE REVIEW

File: en/release-notes.md [hunk 1]
```diff
@@ -118,256 +46,174 @@ within high-availability environments, the configuration of *bind* has been
...
   required an increased number of queries for fields, even when they were not
   being updated. MAAS now tracks specific fields that change and only performs
-  queries for those fields.  For example, to update a power state, MAAS would
+  queries for those fields. For example, to update a power state, MAAS would
   perform 11 queries. After these improvements, only 1 query is now performed.
+- On every transaction, MAAS performed 2 queries to update the timestamp.
+  This has now been consolidated into a single query per transaction.
+- Further reductions in the number of database queries have been made,
+  significantly cutting the queries made by the boot source cache image
+  import process from over 100 to just under 5.  These changes greatly improves
```
Comment (pmatulis) [inline en/release-notes.md]: improve?

File: en/release-notes.md [hunk 2]
```diff
@@ -118,256 +46,174 @@ within high-availability environments, the configuration of *bind* has been
...
 - The node object is updated in the WebSocket only when something has changed
   in the database, reducing the data transferred to the clients as well as the
   amount of internal queries.
+- Only load historic script results (e.g. old commissioning/testing results) when
+  requested / accessed by the user, instead of always making them available over
+  the websocket.
+- Only load node objects in listing pages when the specific object type is
+  requested. For instance, only load machines when accessing the machines tab
+  instead of also loading devices and controllers.
+- Change the UI mechanism to only request OS Information only on initial page
```
Comment (pmatulis) [inline en/release-notes.md]: information? Or is this a header somewhere?

File: en/release-notes.md [hunk 3]
```diff
@@ -118,256 +46,174 @@ within high-availability environments, the configuration of *bind* has been
...
-#### Audit logging
+#### KVM pod improvements
 
-Continuing with the audit logging improvements, *alpha2* now adds audit logging
-for all user actions that affect Hardware Testing & Commissioning.
+KVM pods were initially created to help developers quickly iterate and test new
+functionality while developing MAAS. But KVM pods have also become an essential
+tool for administrators, allowing them to make better use of resources across
+the datacenter. As KVM pods were initially targeted at developers, MAAS lacked
+features more useful for administrators. MAAS 2.4 rectifies these shortcomings
```
Comment (pmatulis) [inline en/release-notes.md]: I feel this paragraph can be lightened up. I find there is too much repetition (developers, developing, administrators).

File: en/release-notes.md [hunk 4]
```diff
@@ -118,256 +46,174 @@ within high-availability environments, the configuration of *bind* has been
...
+  demand based on its AZ. All VMs created from a pod will inherit the AZ.
 
-[https://launchpad.net/maas/+milestone/2.4.0alpha2](https://launchpad.net/maas/+milestone/2.4.0alpha2)
+**Pod tagging**
 
-## 2.4.0 (alpha1)
+- MAAS now adds the ability to set tags for a pod. This allows administrators to use tags
+  to allow/prevent the creation of a VM inside the pod using tags. For example, if the
+  administrator would like a machine with a ‘tag’ named ‘virtual’, MAAS will filter all
+  physical machines and only consider other VM’s or a KVM pod for machine allocation.
```
Comment (pmatulis) [inline en/release-notes.md]: I would write `VMs` but that's just me.

File: en/release-notes.md [hunk 5]
```diff
@@ -118,256 +46,174 @@ within high-availability environments, the configuration of *bind* has been
...
+  different storage pools. This feature uses storage tags to automatically map a
+  storage pool in *libvirt* with a storage tag in MAAS.
 
-### New Features & Improvements
+See the [Pods documentation][pods-docs] for further details.
+
+#### NTP services now provided by Chrony
 
-#### Machine Locking
+In common with changes made to Ubuntu Server, ‘ntpd’ has been replaced with
```
Comment (pmatulis) [inline en/release-notes.md]: Didn't we see this already?

File: en/release-notes.md [hunk 6]
```diff
@@ -118,256 +46,174 @@ within high-availability environments, the configuration of *bind* has been
...
+MAAS adds the ability to lock machines, preventing the user from performing
+actions that could change their state. This gives MAAS a prevention mechanism
+for potentially catastrophic actions. For example, it prevents powering off
+machines by mistake, or releasing machines that could bring workloads down.
 
 #### Audit logging
 
-With the introduction of *audit logging*, MAAS 2.4 allows the administrators to
+With the introduction of audit logging, MAAS 2.4 allows the administrators to
 audit the user’s actions.
```
Comment (pmatulis) [inline en/release-notes.md]: audit users' actions?
audit the actions of users?

File: en/release-notes.md [hunk 7]
```diff
@@ -396,45 +242,108 @@ These improvements allow administrators to:
...
+
+Top-level navigation has now changed to:
+
+- *Zones* renamed to *AZs* (Availability Zones).
+- *Machines*, *Devices* and *Controllers* have been moved from *Hardware* to
+  the top-level menu.
+
+**Settings page reorganisation**
+
+- The web UI MAAS setting pages has been reorganised into tabs, making
```
Comment (pmatulis) [inline en/release-notes.md]: have?

File: en/release-notes.md [hunk 8]
```diff
@@ -396,45 +242,108 @@ These improvements allow administrators to:
...
+more compliant with the Intel IPMI 2.0 spec. Unfortunately, the IPMI 2.0 spec
+has made changes that provide a non-backward compatible user experience. For
+example, if the administrator configures their machine to always PXE boot over
+EFI, and the user executes an IPMI command without specifying the boot type,
+the machine would use the value of the configured BIOS. However, with these new
+changes, the user is required to always specify a boot type, avoiding a
+fallback to the BIOS. 
+
+As such, MAAS now allows the selection of a boot type (auto, legacy, EFI) to
+force the machine to always PXE with the desired type (on the next boot only) .
```
Comment (pmatulis) [inline en/release-notes.md]: PXE-boot?
network-boot?
boot over the network?

**Comment by degville** [comment]
Thanks a lot @pmatulis. Lots of those errors were from me copy pasting from the Google doc. Thanks for looking at this so quickly.

**Pull request merged**